.pre-commit-config.yaml

```diff
@@ -43,6 +43,6 @@ repos:
     -   id: changelogs-rst
         name: changelog filenames
         language: fail
-        entry: 'changelog files must be named ####.(feature|bugfix|doc|removal|vendor|trivial).rst'
+        entry: 'changelog files must be named ####.(feature|bugfix|doc|deprecation|removal|vendor|trivial).rst'
```

Comment:
Oops my bad, missed this one

Reply:
i just noticed it due to a typo in the filename, so it was only a display error, not a technical error ^^